types/aws-sdk2-types/lib/response.d.ts

```diff
@@ -0,0 +1,38 @@
+import {HttpResponse} from './http_response';
+import {Request} from './request';
+export class Response<D, E> {
+		/**
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/aws-sdk2-types/lib/response.d.ts;
  Line: 4;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.